src/styles/header.scss

```diff
@@ -0,0 +1,58 @@
+@import 'variables';
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+
+  .logo {
+    display: flex;
+  }
+
+  .nav__list {
+    list-style-type: none;
+    display: flex;
+    padding: 0;
+    margin: 0;
+
+    .nav__item {
+      margin-right: 20px;
+      position: relative;
+
+      &:last-child {
+        margin-right: 0;
```

Comment:
It's a good practice to avoid setting margins on the BEM element itself. Instead, you could use a modifier or a mix with another class that sets the margin. This makes the BEM block more reusable and prevents potential issues with unwanted margins when the block is used in different contexts.